test/framework/go.mod

```diff
@@ -11,6 +11,5 @@ require (
 	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
 	sigs.k8s.io/cluster-api v0.0.0-00010101000000-000000000000
 	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/cluster-api v0.2.6-0.20191216153338-db9baf9ade0c
```

Comment:
this will just have to be bumped regularly, perhaps even switching to a released version of CAPI in the future